config/uitests-amo.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  apiHost: 'http://olympia.test',
+  proxyApiHost: 'http://olympia.test',
+  proxyPort: 3000,
+  proxyEnabled: true,
+  // Setting this to false returns add-ons that are not compatible but means
+  // developers can pull from a much larger dataset on the local/-dev/-stage
+  // servers. Set this to true to only get compatible add-ons (this is what
+  // happens in production) but get a lot fewer add-ons in search results.
+  restrictSearchResultsToAppVersion: false,
+  fxaConfig: 'default',
```

Comment:
We should probably use the correct value here, i.e. `local`, and get rid of th `FXA_CONFIG` env var in the `package.json`.

Reply:
Well I have to use default for the tests, that is why I am creating this config. I can just set it here though.

Reply:
so, it should be `local` I guess.

Reply:
Do you want `default` or `local` here?